### PR TITLE
[#91453332][3pts] Main menu

### PR DESCRIPTION
**Pull request by zalapeach**
# This pull request involves

* Creates menu style and effects

menu

# Notes

* The main.css file is generated by the sass gem

### CODE REVIEW

**Comment by ingedmundo** [comment]
I think you should use a different name for assets/stylesheets/js.scss a more descriptive one.

File: assets/stylesheets/js.scss [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  -o-transform: translateX(-20px) translateY(20px);
+  -webkit-transform: translateX(-20px) translateY(20px);
+  height: 4px;
+  left: 4%;
+  position: fixed;
+  top: 33px;
+  transform: translateX(-20px) translateY(20px);
+  width: 4px;
+  span {
+    -moz-transform-origin: 50% 50%;
```
Comment (ingedmundo) [inline assets/stylesheets/js.scss]: we are using a couple of bourbon mixins for transform and transition http://bourbon.io/docs/#transform maybe you can use them here

Comment (zalapeach) [inline assets/stylesheets/js.scss]: Thanks, reading that :)

**Comment by ingedmundo** [comment]
LGTM

File: assets/stylesheets/js-menu-styles.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.js-menu-content {
+  background-color: $white;
+  z-index: 1;
+}
+
+.js-menu-nav-overlay {
+  z-index: 2;
+  span { background-color: $breaker-bay; }
```
Comment (ingilniero) [inline assets/stylesheets/js-menu-styles.scss]: What if you use the child selector for this `.js-menu-nav-overlay { > span { ... }  } ` to minify the depth of applicability.

Comment (ingilniero) [inline assets/stylesheets/js-menu-styles.scss]: Beside my comment, it LGTM, nice animation btw.

**Pull request merged**